db/seeds.rb

```diff
@@ -8,10 +8,6 @@
 
 # Employee.create(name: 'Karina Delgado', email: 'karina@hotmail.com', position: 'QA', private_number: 123, status: 'active')
 # Employee.create(name: 'Daniela Sotomayor', email: 'dany@hotmail.com', position: 'Developer', private_number: 321, status: 'active')
-admin = Admin.new(user: 'Karina')
-admin.password = "12345"
-admin.password_confirmation = "12345"
-admin.save
-
+ Admin.create(user: 'admin', password_digest: 'admin')
```

Comment:
Did you verify this seeds to be creating valid users?

Reply:
Yes, we had mentorship for that